gulp/tasks/favicon.js

```diff
@@ -0,0 +1,65 @@
+module.exports = function (gulp, plugins, config) {
+  return function javascript(cb) {
+    plugins.realFavicon.generateFavicon({
+      masterPicture: 'assets/logo.svg',
+      dest: 'assets/favicon',
```

Comment:
i want the icons in the repo.
The API can generate the icons, but needs to be up and is a 3rd-party service.